### PR TITLE
DEEX Integration

### PR DESCRIPTION
**Pull request by estherleongym**
- Added Pairs, Tickers, and updated README for DEEX
- Closes: #925 
- [X] I have added Specs
- [X] (If implementing Market Ticker) I have verified that the `volume` refers to BASE
- [X] (If implementing Market Ticker) I have verified that the `base` and `target` is assigned correctly

### CODE REVIEW

File: lib/cryptoexchange/exchanges/deex/services/order_book.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
+module Cryptoexchange::Exchanges
+  module Binance
```
Comment (wongy91) [inline lib/cryptoexchange/exchanges/deex/services/order_book.rb]: how come binance is here.

Comment (tmlee) [reply lib/cryptoexchange/exchanges/deex/services/order_book.rb]: @estherleongym probably want to fix this?

Comment (kornoholio) [reply lib/cryptoexchange/exchanges/deex/services/order_book.rb]: @estherleongym can i somehow contact you? Will you edit it?

**Comment by estherleongym** [comment]
@tmlee @wongy91 sorry I missed this, I've deleted the file.

**Comment by nick1217** [comment]
Hello everyone! Is there something left undone for the integration?

**Pull request merged**